lib/media/drm_engine.js

```diff
@@ -774,6 +783,48 @@ shaka.media.DrmEngine.prototype.configureClearKey_ = function() {
 };
 
 
+/**
+ * Returns the DrmInfo that is generated by drm configation.
+ * It activates DrmEngine if drm configs have keySystems.
+ * @param {!shakaExtern.Manifest} manifest
+ * @return {(Array.<{object}>|null)}
```

Comment:
This should be: ` * @return {Array.<shakaExtern.DrmInfo>}`

I think `<{object}>` is bad syntax.  I think the `{}` in that context look like a record type.  And you don't need the alternation with null (`|null`) because Arrays are nullable by default.  (To make it non-nullable, add a `!` to the front.)  Finally, the more specific type shakaExtern.DrmInfo should be used, because `Object` is too loose.